server/cmd/server/.env

```diff
@@ -6,3 +6,9 @@ WALLET_PATH="wallet.pem"
 # Wallet's password (e.g.: json password encrypted wallet).
 # Can be left empty for pem wallets
 WALLET_PASSWORD=""
+# MultiversX proxy (e.g.: https://testnet-gateway.multiversx.com)
+MULTIVERSX_PROXY="https://testnet-gateway.multiversx.com"
+# Bridge address on MultiversX to send transactions to
+BRIDGE_SC_ADDRESS="erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"
+# Max retrials to wait for account nonce update after sending bridge txs
+MAX_RETRIALS_WAIT_NONCE=60
```

Comment:
retries instead of retrials? trial is a term used in law

"# Max retrials to wait for account nonce update after sending bridge txs" - is this seconds, rounds, number of times?

Reply:
Right, good catch, refactored everywhere in the project.
Also, added `seconds` to the name variable and comm

Reply:
ok